js/README.md

```diff
@@ -0,0 +1,193 @@
+# How to run
+```
+REMOVEEZ_CONFIG=./config.js INPUT=input OUTPUT=output RUN_ESLINT=1  yarn --cwd ./vendor/ibexa/rector/js transform
+```
+`--cwd` argument should point to directory where JS transform module is installed (for example `./vendor/ibexa/rector/js`).
+
+## Customizable variables
+### REMOVEEZ_CONFIG
+**Required**: no
+
+**Default**: `removeez.config.js`/`none`
+
+Points to configuration file, where you can modify plugins and transformers config. 
+
+### INPUT
+**Required**: no
+
+**Default**: `src/bundle/Resources/public`
+
+Used to point which directory should be parsed.
+
+### OUTPUT
+**Required**: no
+
+**Default**: copied from `INPUT`
+
+Used to point where files should be saved.
+
+### RUN_ESLINT
+**Required**: no
+
+**Default**: 0
+
+Eslint parsing is switched off by default, because older code (below 4.1) was using obsolete eslint config, and in between 4.1 and current version there were different iterations, this package uses `eslint-config-ibexa` in version **1.1.1**
+
+## Configuration file
+If you need to modify default plugins or configuration for rules, put `removeez.config.js` in main directory.
+```
+module.exports = {
+    plugins: (plugins) => {
+        // modify enabled plugins
+
+        return plugins;
+    },
+    config: (config) => {
+        // modify plugins config
+
+        return config;
+    }
+};
+```
+
+## Plugins config
+Example config:
+```
+{
+    "ibexa-rename-string-values": {
+        "ezform-error": "ibexaform-error",
```

Comment:
```suggestion
        "ezform-error": "ibexa-form-error",
```
?

Reply:
Probably classes at both sides are wrong and there should be `ez-form-error` as well, whoops
Unless it's something that came from form-builder, need to recheck